src/pages/3rd-party/index.hbs

```diff
@@ -0,0 +1,94 @@
+---
+title: '3rd party'
+layout: main.hbs
+---
+
+<div class="container">
+
+  <div class="row">
+    <h3 class="col-sm-12">Useful 3rd party libraries</h3>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-12">
+      <p>
+        Here you can find some third-party libraries that can easily extend OpenLayers features.
```

Comment:
Maybe change to something like: "The libraries below extend or integrate well with OpenLayers."